DependencyInjection/Configuration.php

```diff
@@ -628,6 +628,7 @@ private function getOrmEntityManagersNode(): ArrayNodeDefinition
                     ->arrayNode('schema_ignore_classes')
                         ->prototype('scalar')->end()
                     ->end()
+                    ->scalarNode('report_fields_where_declared')->defaultFalse()->end()
```

Comment:
I think a call to `info()` will be quite important for this setting, because it's hard to understand.

Reply:
added